examples/java-native/src/main/java/com/example/myproject/BUILD

```diff
@@ -28,6 +28,18 @@ java_library(
     resources = ["//examples/java-native/src/main/resources:greeting"],
 )
 
+java_binary(
+    name = "hello-special-resources",
+    main_class = "com.example.myproject.Greeter",
+    runtime_deps = [":special%greeting"],
+)
+
+java_library(
+    name = "special%greeting",
```

Comment:
It's a bit awkward to put this special target in our examples. Can you instead move the test to the bazel_windows_test?